webapp/channels/src/components/user_settings_modal/index.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {connect} from 'react-redux';
+
+import {isModalOpen} from 'selectors/views/modals';
+
+import {ModalIdentifiers} from 'utils/constants';
+
+import type {GlobalState} from 'types/store';
+
+import UserSettingsModal from './user_settings_modal';
+
+function mapStateToProps(state: GlobalState) {
```

Comment:
I don't think this `mapStateToProps` is needed because the modal is opened using the modal controller